src/java.base/share/classes/java/lang/Module.java

```diff
@@ -176,6 +177,7 @@ public final class Module implements AnnotatedElement {
      * @see ClassLoader#getUnnamedModule()
      * @jls 7.7.5 Unnamed Modules
      */
+    @ForceInline
```

Comment:
Doesn't the `ensureNativeAccess` code only depend on this one? Also, I'm having an hard time thinking that C2 can't inline this simple method? Isn't this an "accessor" ?